packages/nimble-components/src/mapping/mention-user/index.ts

```diff
@@ -0,0 +1,25 @@
+import { attr } from '@microsoft/fast-element';
```

Comment:
We should just refer to this as `mapping-user`
It's just how nimble will represent a user generally (there is nothing mention specific, this is just user data)
So in the file paths and file contents of mapping-user we should clean out references to mentions

Reply:
Updated as `mapping/user`.